readthedocs/projects/tasks/builds.py

```diff
@@ -728,6 +744,13 @@ def after_return(self, status, retval, task_id, args, kwargs, einfo):
         except Exception:
             log.exception("Failed to revoke build api key.", exc_info=True)
 
+        # Disable scale-in protection on this instance
+        if self.data.project.has_feature(Feature.SCALE_IN_PROTECTION):
+            set_builder_scale_in_protection.delay(
+                instace=socket.gethostname(),
```

Comment:
Oops, sorry I wasn't clear, there is a typo here too though.

Reply:
Meh, you where clear but I think I made a mistake 🙃

Reply:
Fixed